src/minidriver/cardmod-mingw-compat.h

```diff
@@ -0,0 +1,28 @@
+#define __deref
```

Comment:
As I wrote in commit message this file comes from Alon Bar-Lev's old build repository.

This repository ( https://github.com/alonbl/OpenSC-Build ) is marked as GPL-3.0, however
it is debatable that such a small and trivial set of defines warrant a full GPL 3 coverage

They are easy to rewrite from scratch anyway, just remove them and take note of every compiler complaint about
an unknown type / unexpected identifier.